src/librustdoc/markdown.rs

```diff
@@ -166,6 +166,9 @@ pub fn test(input: &str, cfgs: Vec<String>, libs: SearchPaths, externs: Externs,
     old_find_testable_code(&input_str, &mut collector, DUMMY_SP);
     find_testable_code(&input_str, &mut collector, DUMMY_SP);
     test_args.insert(0, "rustdoctest".to_string());
+    if display_warnings {
+        test_args.insert(1, "--display-stdout".to_string());
```

Comment:
Aaaaah. My bad. So basically, in libtest, if I have the `display-stdout` option enabled, for every non-failing test, I display what the test sent to the output.